front-end/assets/css/main.scss

```diff
@@ -0,0 +1,19 @@
+// assets/styles.scss
+$primary-color: #3498db;
```

Comment:
Refactor and move variables to a `_variables.scss` partial.

Reply:
thank you for helping me out. this was done because i wanted to check whether SASS is working as expected or ot. that is only draft ones to quickly check

Reply:
You can experiment and check whether SASS is working or not while also writing clean code at the same time.